src/js/CoreActionConsts.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  "AddCheck": "ADD_CHECK",
+  "Nextverse": "NEW_VERSE",
+  "PrevVerse": "NEW_VERSE",
```

Comment:
A couple thing:
  1. You forgot to capitalize the "V" on line 3.
  2. So does this mean that `NextVerse` and `PrevVerse` are the same action?